src/drivers/boards/px4fmu-v3/px4fmu_init.c

```diff
@@ -119,17 +119,10 @@ __END_DECLS
 /****************************************************************************
  * Private Data
  ****************************************************************************/
-#if defined(BOARD_HAS_SIMPLE_HW_VERSIONING)
-static int hw_version = 0;
-static int hw_revision = 0;
+static int hw_version = -1;
```

Comment:
@dagar - there was an API change.
```
* Returned Value:
 *   An integer value of this boards hardware version.
 *   A value of -1 is the default for boards not supporting the BOARD_HAS_VERSIONING API.
 *   A value of 0 is the default for boards supporting the API but not having version.
```

Reply:
Ah good catch. I went back to master and redid these with unifdef.

Reply:
Fixed